apps/chatbot/src/app/main.py

```diff
@@ -18,30 +18,30 @@
 
 params = yaml.safe_load(open("config/params.yaml", "r"))
 prompts = yaml.safe_load(open("config/prompts.yaml", "r"))
-chatbot = Chatbot(params, prompts)
-
 AWS_DEFAULT_REGION = os.getenv('CHB_AWS_DEFAULT_REGION', os.getenv('AWS_DEFAULT_REGION', None))
 
+chatbot = Chatbot(params, prompts)
+
 
 class Query(BaseModel):
   question: str
   queriedAt: str | None = None
 
-if (os.getenv('environment', 'dev') == 'local'):
-  profile_name='dummy'
```

Comment:
I saw this `dummy` in many files. isn't better to but this `profile_name` in `config/params.yaml` or in a enviroment variable?